gflownet/envs/spacegroup.py

```diff
@@ -7,27 +7,28 @@
 import numpy as np
 import torch
 from torch import Tensor
+from torchtyping import TensorType
 
 from gflownet.envs.base import GFlowNetEnv
-from gflownet.utils.crystals.constants import (
-    CRYSTAL_SYSTEMS,
-    CRYSTAL_CLASSES,
-    POINT_SYMMETRIES,
-    SPACE_GROUPS,
-)
+from gflownet.utils.crystals.constants import (CRYSTAL_CLASSES,
```

Comment:
Wait, did black actually format this or is it manual? Mine seems to keep the previous version, and I am worried about it reverting it on next modification.

Reply:
ugh this is an annoying conflict between `isort` and `black`. I guess I can fix it with some configuration by I haven't got my head around it yet.

Reply:
Could this help (disclaimer: I didn't test it)? https://github.com/PyCQA/isort/issues/1518

Reply:
Works! The latest commit does
```
black .
```
then
```
isort --profile black .
```